app-creator/src/widgets/empty-notice/empty-notice.ts

```diff
@@ -0,0 +1,110 @@
+/**
```

Comment:
Thought it would be better to create an empty notice widget since we use it in multiple places